pkg/webhook/namespace/patch.go

```diff
@@ -72,11 +72,6 @@ func (r *patchHandler) OnUpdate(c client.Client, decoder *admission.Decoder, rec
 
 				return &response
 			}
-		} else {
-			recorder.Eventf(ns, corev1.EventTypeWarning, "NamespacePatch", e)
-			response := admission.Denied(e)
-
-			return &response
```

Comment:
I'm a bit concerned about removing this, may I ask you for some support in getting the reason?

Reply:
@prometherion As @sagar-jadhav stated above non-capsule managed namespaces will not have this label, in that case, this else will always block. Is there any use case for which this is needed?

Reply:
You're right, thanks for pointing it out!